src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java

```diff
@@ -4,34 +4,24 @@
 import com.amazonaws.regions.Region;
 import com.amazonaws.regions.RegionUtils;
 import com.amazonaws.services.cloudwatch.AmazonCloudWatchClient;
-import com.amazonaws.services.cloudwatch.model.Datapoint;
-import com.amazonaws.services.cloudwatch.model.Dimension;
-import com.amazonaws.services.cloudwatch.model.DimensionFilter;
-import com.amazonaws.services.cloudwatch.model.GetMetricStatisticsRequest;
-import com.amazonaws.services.cloudwatch.model.GetMetricStatisticsResult;
-import com.amazonaws.services.cloudwatch.model.ListMetricsRequest;
-import com.amazonaws.services.cloudwatch.model.ListMetricsResult;
-import com.amazonaws.services.cloudwatch.model.Metric;
+import com.amazonaws.services.cloudwatch.model.*;
 import io.prometheus.client.Collector;
 import io.prometheus.client.Counter;
+import org.yaml.snakeyaml.Yaml;
 
 import java.io.FileReader;
-import java.io.Reader;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Date;
-import java.util.List;
-import java.util.Set;
-import java.util.Map;
-import java.util.HashMap;
+import java.io.Reader;
+import java.util.*;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import java.util.regex.Pattern;
 
-import org.yaml.snakeyaml.Yaml;
+import com.google.common.collect.Lists;
```

Comment:
Do we really need this?

Reply:
We use it for exactly one call - partition a list into chunks of 100 elements. This is the maximum queries per GetMetricData request. I think it is convenient not to implement this ad hoc with a for loop and if statement.

Reply:
I'd rather not pull in a new dependency for the sake of a few lines of code.